app/topics/shared/topic.model.ts

```diff
@@ -122,6 +148,9 @@ export class Topic {
     this.createdAt = createdAt;
     this.subTopics = subTopics;
     this.parentTopics = parentTopics;
+    this.studentID = studentID;
+    this.supervisorID = supervisorID;
+    this.reviewerID = reviewerID;
```

Comment:
When using number array, the ids should not be part of an constructor, but computed by looping through the UserArray